bin/xbundle_convert

```diff
@@ -1,69 +1,64 @@
 #!/usr/bin/env python
 
-"""
-A command-line script to use xbundle.
+"""xbundle_convert
+
+Description:
+    Convert between OLX and xbundle XML formats.
+    If the input format in an XML file, the output will be OLX.
+    If the input format is a directory, the output will be xbundle.
+
+Usage:
+    xbundle_convert convert [--force-studio] <input> <output>
+    xbundle_convert test
+    xbundle_convert --help
+    xbundle_convert -h
+
+Options:
+    --force-studio  Forces <sequential> to be followed by <vertical> in export
 """
 
-import sys
+# PyPi
+from docopt import docopt
 
+# local
 from xbundle import XBundle, run_tests
 
 # pylint: disable=star-args
 
-def usage():
-    """
-    print usage
-    """
-    print "Usage: python xbundle.py [--force-studio] [cmd] [infn] [outfn]"
-    print "where:"
-    print "  cmd = test:    run unit tests"
-    print "  cmd = convert: convert between xbundle and edX directory format"
-    print "                 the xbundle filename must end with .xml"
-    print "  --force-studio forces <sequential> to always be followed by <vertical> in export"
-    print "                 this makes it compatible with Studio import"
-    print ""
-    print "examples:"
-    print "  python xbundle.py convert ../data/edx4edx edx4edx_xbundle.xml"
-    print "  python xbundle.py convert edx4edx_xbundle.xml ./"
-
 def main():
     """
     Convert between OLX and xbundle XML formats.
     """
-    if len(sys.argv) < 2:
-        usage()
-        sys.exit(0)
 
-    argc = 1
+    # This parses the docstring above and derives all arguments.
+    # If the required arguments are missing, it prints basic usage
+    # and exits automatically.
+    # If required arguments are satisfied, it returns a dict.
+    args = docopt(__doc__)
```

Comment:
It looks like it's not such an easy problem, considering that Flask does it with ast, running an eval on __init__.py to do it. https://github.com/mitsuhiko/flask/blob/master/setup.py

I tried something simpler, but ran into a dependency issue (importing from __init__.py in setup.py fails because lxml isn't installed). Same when `VERSION` is moved into its own file. And putting `VERSION` in the directory with `setup.py` means that the command-line tool won't have access to it.

I'm considering just doing a `with open...` like Flask, minus the ast and regex, and just reading the string.